Submodels/Parameterizations.py

```
@@ -414,26 +498,174 @@ def ConvertBezier2AirfoilCoordinates(self,
-        lower_x = camber_x + thickness_distribution * np.sin(theta)
-        lower_y = camber - thickness_distribution * np.cos(theta)
+        else:
+            upper_x = thickness_x
+            upper_y = thickness
+            lower_x = thickness_x
+            lower_y = -thickness
 
         return upper_x, upper_y, lower_x, lower_y
 
 
+    def ComputeBezierCurves(self,
```

Ruff (0.8.2)

Missing return type annotation for public function `ComputeBezierCurves`

(ANN201)